src/Droppable/Droppable.js

```diff
@@ -140,6 +141,18 @@ export default class Droppable extends Draggable {
       return;
```

Comment:
Note to self: this needs a test ^

`it('prevents dragging of draggable element without droppable parent')`